swssconfig/sample/sample.json

```diff
@@ -51,4 +51,14 @@
 			},
 			"OP": "SET"
 		}
-	]
\ No newline at end of file
+		{
```

Comment:
can you put this as a separate sample file.

for example, netbouncer.json

also for the example output, put it as a separate file netbouncer.json.output